applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp

```diff
@@ -0,0 +1,2801 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Peter Wilson
+//       contact:    A.Winterstein@tum.de
+//
+
+#include "shell_thick_element_3D3N.hpp"
+#include "custom_utilities/shellt3_corotational_coordinate_transformation.hpp"
+#include "structural_mechanics_application_variables.h"
+
+#include "custom_constitutive/linear_elastic_orthotropic_2D_law.hpp"
+
+#include "geometries/triangle_3d_3.h"
+
+#include <string>
+#include <iomanip>
+
+#define OPT_NUM_NODES 3
```

Comment:
Can you give an example of how to define it locally? This is copy-paste from the existing shells